pom.xml

```diff
@@ -517,20 +495,6 @@
           <encoding>${project.build.sourceEncoding}</encoding>
         </configuration>
       </plugin>
-      <plugin>
```

Comment:
Keep this one removed as this was causing the issue with mvn:site and we don't use this in any downstream projects so it won't have any impact.

Reply:
done, thanks